src/webhook/handlers/__tests__/askingArticleSubmissionConsent.test.js

```diff
@@ -34,10 +34,9 @@ it('should block incorrect prefix', async () => {
       searchedText: 'Some text forwarded by the user',
       foundArticleIds: [],
     },
-    state: 'ASKING_ARTICLE_SUBMISSION_CONSENT',
     event: {
       type: 'message',
-      input: REASON_PREFIX + 'foo', // Wrong prefix
+      input: SOURCE_PREFIX_NOT_YET_REPLIED + 'foo', // Wrong prefix
```

Comment:
I am wondering if it is possible for `should block incorrect prefix` to happen under the current action handling logic, since `SOURCE_PREFIX_NOT_YET_REPLIED` will be redirected to other state for now.

If not, I think we can remove the logic that emits `ManipulationError` together.